src/core/Form/RadioButton/RadioButton.tsx

```diff
@@ -1,74 +1,193 @@
-import React, { Component } from 'react';
+import React, { Component, ReactNode } from 'react';
 import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlInput, HtmlLabel, HtmlSpan, HtmlDiv } from '../../../reset';
 import { TokensProp, InternalTokensProp } from '../../theme';
-import {
-  RadioButton as CompRadioButton,
-  RadioButtonProps as CompRadioButtonProps,
-} from '../../../components/Form/RadioButton';
-import { RadioButtonGroup, RadioButtonGroupProps } from './RadioButtonGroup';
-import {
-  RadioButtonDivider,
-  RadioButtonDividerProps,
-} from './RadioButtonDivider';
-import { baseStyles } from './RadioButton.baseStyles';
+import { logger } from '../../../utils/logger';
 import { withSuomifiDefaultProps } from '../../theme/utils';
-import classnames from 'classnames';
+import { AutoId } from '../../../utils/AutoId';
+import { getConditionalAriaProp } from '../../../utils/aria';
+import { RadioButtonGroupConsumer } from './RadioButtonGroup';
+import { baseStyles } from './RadioButton.baseStyles';
 
 const baseClassName = 'fi-radio-button';
 const radioButtonClassNames = {
+  container: `${baseClassName}_container`,
+  input: `${baseClassName}_input`,
+  label: `${baseClassName}_label`,
+  hintText: `${baseClassName}_hintText`,
   disabled: `${baseClassName}--disabled`,
   large: `${baseClassName}--large`,
   checked: `${baseClassName}--checked`,
 };
 
-export interface RadioButtonProps extends CompRadioButtonProps, TokensProp {}
+type RadioButtonVariant = 'small' | 'large';
```

Comment:
Is there need to export this or why this is a separate type? 
As earlier discussed with previous PR's we thought that it union type could just be in the interface if there is no certain need to have it out.

Reply:
Might be better to just move it to the interface so there would be no need to export it.

Reply:
Changed